client/tx/tx.go

```diff
@@ -56,7 +56,7 @@ func GenerateTx(clientCtx client.Context, txf Factory, msgs ...sdk.Msg) error {
 		}
 
 		txf = txf.WithGas(adjusted)
-		_, _ = fmt.Fprintf(os.Stderr, "%s\n", GasEstimateResponse{GasEstimate: txf.Gas()})
+		_, _ = fmt.Fprintf(clientCtx.Output, "%s\n", GasEstimateResponse{GasEstimate: txf.Gas()})
```

Comment:
@aaronc we could add an `Error io.Writer` field to `client.Context`. Wdyt?

Reply:
Sure, but why does this go to error to begin with? Also why can't we just run tests off Output?

Reply:
Yeah I'm not against printing the output to stdout. When you use the `--dry-run` flag you really want the gas estimate to be printed out. But then, in that case, we should not print anything other than the gas estimate, if that makes sense.

Reply:
We need a CLI product owner to make these UX decisions.

Reply:
> We need a CLI product owner to make these UX decisions.

I've been doing that all along, in fact, one of my first tasks, when I was hired at AiB, was to overhaul the command line interfaces (I believe @jackzampolin can testify that).

Happy to hand over the baton to someone or even hand over the baton to someone else. As you see fit, really.

Reply:
If you're the CLI product owner then glad to know! 😄

Reply:
Can you make a call here on what is best @alessio ?